### ISSUE TITLE
'seaborn-whitegrid' is not a valid package style

### ISSUE THREAD

**Issue by ivanpetrushev** [arundo/adtk#152]
I see `plot()` explicitly sets the `seaborn-whitegrid` as a plot style: https://github.com/arundo/adtk/blob/master/src/adtk/visualization/_visualization.py#L201

Is it possible to have the plot style set as a configuration option, or leave it entirely out of this codebase?

The problem is that I can't get `seaborn-whitegrid` to register in my environment at all.

requirements.txt:
```
numpy==1.26.1
scipy==1.11.3
influxdb-client==1.36.1
pandas==2.1.2
matplotlib==3.8.1
adtk==0.6.2
SciencePlots==2.1.0
seaborn==0.13.0
```

plt.style.available:
```
['Solarize_Light2', '_classic_test_patch', '_mpl-gallery', '_mpl-gallery-nogrid', 'bmh', 'classic', 'dark_background', 'fast', 'fivethirtyeight', 'ggplot', 'grayscale', 'seaborn-v0_8', 'seaborn-v0_8-bright', 'seaborn-v0_8-colorblind', 'seaborn-v0_8-dark', 'seaborn-v0_8-dark-palette', 'seaborn-v0_8-darkgrid', 'seaborn-v0_8-deep', 'seaborn-v0_8-muted', 'seaborn-v0_8-notebook', 'seaborn-v0_8-paper', 'seaborn-v0_8-pastel', 'seaborn-v0_8-poster', 'seaborn-v0_8-talk', 'seaborn-v0_8-ticks', 'seaborn-v0_8-white', 'seaborn-v0_8-whitegrid', 'tableau-colorblind10']
```

Somehow I have `seaborn-v0_8-whitegrid` available, but no `seaborn-whitegrid`.

Environment: docker image FROM public.ecr.aws/lambda/python:3.10

**Comment by nabeelraza-7** [comment]
I was getting the same issue so I forked and removed the styling entirely, [here](https://github.com/nabeelraza-7/adtk). If you want to style them, you can set the styles yourself.

**Comment by BreadFeet** [comment]
True, it is not one of the styles available, following this official document: https://matplotlib.org/stable/gallery/style_sheets/style_sheets_reference.html

Hope it will get fixed sooner or later!

**Comment by gbakerpennsy** [comment]
I just went into ...\mpl-data\stylelib and copied over seaborn_v8... to 'seaborn-whitegrid'
remember to reload matplotlib

**Comment by Datakriger101** [comment]
> I just went into ...\mpl-data\stylelib and copied over seaborn_v8... to 'seaborn-whitegrid' remember to reload matplotlib

Worked like a charm. Thank you

**Comment by csrgxtu** [comment]
i met this problem 2

**Comment by csrgxtu** [comment]
https://github.com/arundo/adtk/pull/156

**Comment by alyakoubiaous** [comment]
same problem here
how did u guys solve it ?

**Comment by dam890** [comment]
Checked available styles:
import matplotlib.pyplot as plt
print(plt.style.available)

Replaced style:
plt.style.use('seaborn-whitegrid')

To style:
plt.style.use('seaborn-v0_8-whitegrid')

works)

**Comment by joserezende** [comment]
> Checked available styles: import matplotlib.pyplot as plt print(plt.style.available)
> 
> Replaced style: plt.style.use('seaborn-whitegrid')
> 
> To style: plt.style.use('seaborn-v0_8-whitegrid')
> 
> works)

It didn't work for me. I'm using VSCode from Anaconda Navigator.

**Comment by joserezende** [comment]
> I just went into ...\mpl-data\stylelib and copied over seaborn_v8... to 'seaborn-whitegrid' remember to reload matplotlib


It didn't work for me. How to reload matplotlib? I'm using VSCode from Anaconda Navigator.

**Comment by esaostring** [comment]
I stumbled upon this also. I believe you'd have to remove corresponding `__pycache__` -directory and its contents after editing the `_visualization.py` file.
